nx_cugraph/drawing/layout.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this is to prevent ruff from complaining about newlines in docstrings
+# the docstring should show up properly in the nx docs
+# ruff: noqa: D205
+
+import cupy as cp
+import networkx as nx
+import pylibcugraph as plc
+
+from nx_cugraph.convert import _to_graph
+from nx_cugraph.utils import (
+    _dtype_param,
+    _get_float_dtype,
+    _seed_to_int,
+    networkx_algorithm,
+)
+
+__all__ = [
+    "forceatlas2_layout",
+]
+
+
+@networkx_algorithm(
+    extra_params={
+        "outbound_attraction_distribution : bool, default True": (
+            "Distributes attraction along outbound edges. "
+            "Hubs attract less and thus are pushed to the borders."
+        ),
+        **_dtype_param,
+    },
+    version_added="25.04",
+    _plc="forceatlas2_layout",
+)
+def forceatlas2_layout(
+    G,
+    pos=None,
+    *,
+    max_iter=100,
+    jitter_tolerance=1.0,
+    scaling_ratio=2.0,
+    gravity=1.0,
+    distributed_action=False,
+    strong_gravity=False,
+    node_mass=None,
+    node_size=None,
+    weight=None,
+    dissuade_hubs=False,
+    linlog=False,
+    seed=None,
+    dim=2,
+    store_pos_as=None,
+    # nx_cugraph-only argument
+    outbound_attraction_distribution=True,
+    dtype=None,
+):
+    """
+    `distributed_action`, `node_mass`, `node_size` parameter is currently ignored.
+    Only `dim=2` is supported. `outbound_attraction_distribution` is
+    a backend-only argument.
```

Comment:
I think there's no need to mention backend-only arguments, since they will be documented next.
```suggestion
    `distributed_action`, `node_mass`, and `node_size` parameters are currently ignored.
    Only `dim=2` is supported.
```
![FA2](https://github.com/user-attachments/assets/63d0b823-1ded-4018-a434-4ea1fb86db72)